ft_memdel.c

```diff
@@ -0,0 +1,22 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_memdel.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: ekorley <marvin@42.fr>                     +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2016/09/22 17:37:50 by ekorley           #+#    #+#             */
+/*   Updated: 2016/09/22 17:43:41 by ekorley          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "libft.h"
+
+void	ft_memdel(void **ap)
+{
+	void	*ptr;
+
+	ptr = *ap;
+	free(ptr);
+	ap = NULL;
+}
```

Comment:
I'm not familiar with this function, but if you can wait on the review I'll get back to you.